index.js

```diff
@@ -32,7 +32,17 @@
  */
 
 // Cherry-pick lodash components to help with size
-var _ = require('lodash');
+var _each = require('lodash/each');
```

Comment:
Do you mind stuffing these into an object?  The way I use to do it was like this:

```js
var _ = {
  each: require('lodash/each'),
  // ...
}
```

The reason this is preferred is because I can continue using `_` as if I weren't cherry-picking the functions.